portal/styles/footer.scss

```diff
@@ -0,0 +1,3 @@
+.email-wrap {
+  word-break: break-all;
```

Comment:
This should be word break safe so something like this would be ideal in my opinion.

`word-break: break-word; white-space: break-spaces;`

Reply:
done